state/StateGetBundle.F90

```diff
@@ -1,22 +1,22 @@
-#include "MAPL_Generic.h"
+#include "MAPL_ErrLog.h"
 
-module mapl3g_StateGet
+module mapl3g_StateGetBundle
 
    use mapl_ErrorHandling
    use esmf
 
    implicit none
    private
 
-   public :: MAPL_StateGet
+   public :: StateGetBundle
```

Comment:
We can discuss, but I think we want the interface to simply be `StateGet`, and `bundle` will just be one of the optional arguments.

See `ESMF_StateGet` for reference.

Reply:
Yes, it'll be better to have a `StateGet` with `bundle` being an optional argument. I'll make that change.

Reply:
Done.

Reply:
This has been moved to `field_bundle/FieldBundleCreate.F90`